Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```
@@ -5724,6 +5724,20 @@ ScrollAnimatorEnabled:
     WebCore:
       default: false
 
+ScrollDrivenAnimationsEnabled:
+  type: bool
+  status: unstable
+  category: animation
+  humanReadableName: "Scroll-driven Animations"
+  humanReadableDescription: "Support for the Scroll-driven Animations specification"
```

```suggestion
  humanReadableDescription: "Enable CSS Scroll-driven Animations"
```

In general the description is prefixed with "Enable"

---

I won't add the "CSS" part since this also encompasses a JS API.